File: 기본 수학/1011/1011.c
#include <stdio.h>

unsigned int fly(unsigned int, unsigned int);

int main()
{
   unsigned int n, a, b, i;
   scanf("%d", &n);
   for (i = 0; i < n; ++i)
   {
      scanf("%d %d", &a, &b);
      printf("%d\n", fly(a, b));
   }

   return 0;
}

unsigned int fly(unsigned int a, unsigned int b)
{
   unsigned int n = b - a;
   unsigned int i, check = 0;

   for (i = 0; check < n; check += (i + 1) / 2)
      ++i;

   return i;
}

File: 기본 수학/1011/1011.py
def fly(a, b):
   n = b - a
   check = 0
   i = 0

   while check < n:
      i += 1
      check += (i + 1) // 2

   return i

n = int(input())

for i in range(n):
   a, b = map(int, input().split())
   print(fly(a, b))